src/anim/evaluator/anim-track.js

```diff
@@ -28,26 +27,57 @@ class AnimTrack {
         this._animEvents = animEvents;
     }
 
+    /**
+     * Returns the name of the AnimTrack.
```

Comment:
Can you change Returns to Gets? Since it's a getter (property), it probably fits better. Same applies to the other getters in this PR.

Reply:
It could just be `The name of the AnimTrack.` I guess.....but prepending 'Gets' hints that it's read-only.